### ISSUE TITLE
[Issue][<MazPhoneNumberInput>] Cannot find module

### ISSUE THREAD

**Issue by Temepest74** [LouisMazel/maz-ui#612]
**Describe the bug**
Inside vs and when I run vue-tsc && vite build I get

```
resources/js/Components/InputPhone.vue:2:33 - error TS2307: Cannot find module 'maz-ui/components/MazPhoneNumberInput' or its corresponding type declarations.

2 import MazPhoneNumberInput from 'maz-ui/components/MazPhoneNumberInput';
                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in resources/js/Components/InputPhone.vue:2
```

**To Reproduce**

Just follow the docs for 3.10 and 3.11, everything works great in 3.9. Don't globally import the components

```js
<script setup lang="ts">
import MazPhoneNumberInput from 'maz-ui/components/MazPhoneNumberInput';

const props = defineProps<{
    modelValue: string | undefined;
}>();
defineEmits(['update:modelValue']);
</script>

<template>
    <MazPhoneNumberInput :model-value="modelValue" @update:model-value="$emit('update:modelValue', $event)" />
</template>

```

- OS: WSL
- Version 2

**Comment by LouisMazel** [comment]
Hi @Temepest74,

Bug fixed in **3.11.3**, let me know if it's good for you

Enjoy

**Comment by Temepest74** [comment]
Sorry, I didn't had time yesterdays. This works great now. Thank you

**Comment by MartinX3** [comment]
@LouisMazel Sadly since v3.48.0 it's back.

`Vue: Cannot find module maz-ui/ components/ MazPhoneNumberInput or its corresponding type declarations.`

```
/ path/ node_modules/.pnpm/ maz-ui@3.48.1_libphonenumber-js@1.11.15_vue-router@4.5.0_vue@3.5.13_typescript@5.6.3___vue@3.5.13_typescript@5.6.3_/ node_modules/ maz-ui/ components/ MazPhoneNumberInput. mjs
```

**Comment by LouisMazel** [comment]
Thanks @MartinX3!

It's now fixed in [v3.48.2](https://github.com/LouisMazel/maz-ui/releases/tag/v3.48.2)

Sorry about that
